configure.ac

```diff
@@ -272,9 +272,11 @@ AC_SUBST(QBT_REMOVE_CONFIG)
 AC_SUBST(QBT_ADD_DEFINES)
 AC_SUBST(QBT_REMOVE_DEFINES)
 
-AC_OUTPUT(conf.pri)
+AC_CONFIG_FILES(conf.pri)
+AC_OUTPUT
 AS_IF([test "x$enable_systemd" = "xyes"],
-      [AC_OUTPUT(dist/unix/systemd/qbittorrent-nox@.service)])
+      [AC_CONFIG_FILES(dist/unix/systemd/qbittorrent-nox@.service)
+      AC_OUTPUT])
 
```

Comment:
Why not call `AC_OUTPUT` once and unconditionally in the end?

Reply:
>Why not call AC_OUTPUT once and unconditionally in the end?

Of course, done.